src/__tests__/index.test.tsx

```diff
@@ -1,17 +1,24 @@
 import { NativeModules, Platform } from 'react-native';
-import Chargebee, { SubscriptionsRequest } from '../index';
+import Chargebee, {
+  ProductIdentifiersRequest,
+  SubscriptionsRequest,
+} from '../index';
 
 describe('Chargebee React Native', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
   describe('Configuring Chargebee React Native SDK with Site, Publishable API key and SDK Key', () => {
     const testSite = 'testsite';
     const testPublishableApiKey = 'test-PublishableApiKey';
     const androidSdkKey = 'androidSdkKey';
     const iOsSdkKey = 'iOsSdkKey';
 
-    it('for Android', () => {
+    it('for Android', async () => {
       Platform.OS = 'android';
 
-      Chargebee.configure({
+      await Chargebee.configure({
```

Comment:
Maybe not for this PR, but can we have tests to assert errors thrown from `configure` as well?